packages/app/e2e/config.e2e.js

```diff
@@ -40,11 +40,12 @@ describe('config', () => {
     });
 
     // NOTE: "preferencesClearAll" clears Firestore settings. Set DB as emulator again.
-    after(async () => {
-      await firebase
-        .firestore()
-        .settings({ host: 'localhost:8080', ssl: false, persistence: true });
-    });
+    // FIXME if we use the emulator, then we never throw errors because firestore rules are permissive, that causes test failure
```

Comment:
I'm wary of not using the emulator, this should be working, the rules are defined here and have been working previously:

https://github.com/invertase/react-native-firebase/blob/master/.github/workflows/scripts/firestore.rules

Might be a case of not starting it correctly if you were seeing this issue locally. Should be started as follows:

`cd ./.github/workflows/scripts && sh ./start-firestore-emulator.sh`

Reply:
I agree, I really want the emulator in (I want more emulator, really...auth) and just couldn't find the rules in order to fuzz-bust it. I also got a pointer from Russell, and will work through this bit

Reply:
Starting it with the wrong working directory, so the rules didn't apply, was the ticket.

I have this working now locally with the emulator.

I will add a commit that cleans up emulator start so it's all in package.json (centralized) with either a background mode (for CI) or foreground mode (for local dev) so that this won't happen again and when we add non-firestore emulator capabilities (like auth) we only have to edit one place. 💯